src/net/body.rs

```diff
@@ -357,12 +357,12 @@ enum FileField_ {
 }
 
 trait StreamField: Read + Send + 'static {
-    fn add_self(self: Box<Self>, name: String, filename: Option<String>, content_type: Option<Mime>, to: &mut Multipart);
+    fn add_self(self: Self, name: String, filename: Option<String>, content_type: Option<Mime>, to: &mut Multipart);
```

Comment:
I'm surprised that this works without breaking `Box<StreamField>`, but the build doesn't lie.